lib/ecb_rates/application.rb

```diff
@@ -0,0 +1,28 @@
+require 'date'
+
+module EcbRates
+  class Application
```

Comment:
I mean this part:

```ruby
EcbRates.exchange_rate('JPY')
```
instead of:

```ruby
app = EcbRates::Application.new
app.exchange_rates('JPY')
```

and yeah youre right, exceptions should be moved to EcbRates module.